Copyright text in README pages

kaitlinnewson opened the pull request:
This draft PR will add the copyright text to the index and README pages located in a language directory for a guide, e.g. `learning-ojs/en/README.md`.

The main issue I could use input on is the translated guides, as some of them have translated the copyright text (e.g. https://docs.pkp.sfu.ca/learning-ojs/fr/). I haven't removed these translations yet, but doing so will mean that the english text will appear in these guides instead. Does it make sense to do this, then we can incorporate the translated versions later when we have a fully translated docs hub? Or should we leave it as-is for now?

NateWr commented:
This looks ok to me. I notice that it also adds the copyright statement to the language indexes, but presumably that's ok?

kaitlinnewson commented:
Currently the statement is added to any page that has `isBookIndex: true`, and this PR would expand that to both the `index.md` and `README.md` pages. But we could remove it from the language indexes if that would make more sense.

NateWr commented:
Yeah I don't know, what do you think? I guess it's no harm having it added to the language index.

kaitlinnewson commented:
I've changed it to not show on the language index pages, as I think that makes sense and it will make those pages much cleaner. I also removed the translated copyright text in the translated guides.
